util/lambda_trigger/lambda_trigger.go

```diff
@@ -33,6 +33,18 @@ func isProductSupported(product string) bool {
 	return false
 }
 
+func isCask(product string) bool {
+	casks := []string{"boundary-desktop"}
```

Comment:
Yeah I'm a bit torn on this part.  The lambda function here is what triggers the workflow and therefore the new cli in this PR, and it's meant to be a gatekeeper of whitelisted products to trigger on.  The reason for separating out casks and the existing "supportedProducts" here is to include the new `cask` attribute on the event.

The reason I'm torn is because the way I designed the cli is it's supposed to be indifferent of the destination of the formula or cask.  So you ask it to give the resulting formula or cask for a product based on the config, and it'll give it to you in stdout.  This gives a lot of flexibility in where we can execute this from and how it's used.  It does mean, however, that the workflow executing the cli needs to know where to put the output. Formulae and Casks are stored in different directories, so the `cask` attribute on the event lets the workflow branch to determine where the output of the cli should be saved.

Hopefully that rambling makes sense

Reply:
Hm, yeah I can see writing to stdout is really flexible, maybe once this is ported over to the other tool we could think about a flag to just write the file to its default location as well to save the caller having to be aware?